File: mobile/src/main/java/com/windscribe/mobile/confirmemail/ConfirmEmailView.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.mobile.confirmemail;

public interface ConfirmEmailView {

    void finishActivity();

    void setReasonToConfirmEmail(String reasonForConfirmEmail);

    void showEmailConfirmProgress(boolean show);

    void showToast(String toast);
}

File: mobile/src/main/java/com/windscribe/mobile/confirmemail/ConfirmEmailView.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */
package com.windscribe.mobile.confirmemail

interface ConfirmEmailView {
    fun finishActivity()
    fun setReasonToConfirmEmail(reasonForConfirmEmail: String)
    fun showEmailConfirmProgress(show: Boolean)
    fun showToast(toast: String)
}